### PR TITLE
Add The Meaning Generator

### PR DESCRIPTION
**Pull request by ente3000**
I sure hope I did this right.

### CODE REVIEW

File: _posts/2018-04-19-meaning-generator.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+image: meaning-generator.png
+link: http://meaning-generator.com/
+authorName: Eric Rösch
+authorUrl: https://rsch.neocities.org
+authorGithub: ente3000
+imgWidth: 1920
+imgHeight: 949
+---
+
+_Civilization emerges from loss. Nothing heals memory. Make your meaning._
```
Comment (Synj24) [inline _posts/2018-04-19-meaning-generator.md]: need to add 
```
[The Meaning Generator](http://meaning-generator.com/) - by [Osk Studio](http://osk.co/)
```
to line 14 :slightly_smiling_face:

File: _posts/2018-04-19-meaning-generator.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: The Meaning Generator
```
Comment (Synj24) [inline _posts/2018-04-19-meaning-generator.md]: You might want to add a [`type`](https://github.com/tholman/inspiring-online/wiki/Types-of-Posts) below here aswell. like this: 
```
type: Beautiful
```

File: _posts/2018-04-19-meaning-generator.md (hunk 1 above)
Comment (Synj24) [inline _posts/2018-04-19-meaning-generator.md]: need to add 
```
[The Meaning Generator](http://meaning-generator.com/) - by [Osk Studio](http://osk.co/)
```
to line 14 :slightly_smiling_face:

**Comment by ente3000** [comment]
Alright, done! I made the screenshot 418px in width, just in case.

File: _posts/2018-04-19-meaning-generator.md (hunk 2 above)
Comment (ente3000) [reply _posts/2018-04-19-meaning-generator.md]: Is there, like, a list of types I can use?

Comment (Synj24) [reply _posts/2018-04-19-meaning-generator.md]: Yup Here you go:
https://github.com/tholman/inspiring-online/wiki/Types-of-Posts

Comment (ente3000) [reply _posts/2018-04-19-meaning-generator.md]: 👌

**Comment by TJScalzo** [comment]
This looks great!

**Pull request merged**